### ISSUE TITLE
how should the controller connected to robot

### ISSUE THREAD

**Issue by keweiang** [artrmz/flutter_control_pad#33]
may I know how should the Arduino BLE controller (control pad) be connected to the robot? can I get the code?

**Comment by yassinedevop** [comment]
> may I know how should the Arduino BLE controller (control pad) be connected to the robot? can I get the code?

Really ?! you should ask if you enconter a problem and not if you want us to do your project for you

**Comment by ramuno4** [comment]
I completely agree with @yassinedevop  and state again that this is not the place to ask for such help at all, however, I have used this library to create exactly this [app](https://github.com/Jon1105/bluetooth_controller) (_Not very polished at all_). For the arduino code, connect to a bluetooth module (HC-05) via Serial and use Serial.read() twice or 3 times in a row depending on which page you are using.
